lib/linguist/languages.yml

```diff
@@ -2868,6 +2868,16 @@ ImageJ Macro:
   ace_mode: text
   tm_scope: none
   language_id: 575143428
+Imba:
+  type: programming
+  color: "#16cec6"
+  extensions:
+  - ".imba"
+  ace_mode: text
+  tm_scope: source.imba
+  interpreters:
+  - imba
```

Comment:
I don't think we need this. Interpreters are for helping identify the language of files that use shebangs, and from a quick search it doesn't appear Imba uses shebangs.

```suggestion
```